src/lib/sql/sql_translator.cpp

```diff
@@ -925,6 +938,46 @@ std::shared_ptr<AbstractLQPNode> SQLTranslator::_translate_show(const hsql::Show
   return {};
 }
 
+std::shared_ptr<AbstractLQPNode> SQLTranslator::_translate_create(const hsql::CreateStatement& create_statement) {
+  switch (create_statement.type) {
+    case hsql::CreateType::kCreateView: {
+      auto view = _translate_select((const hsql::SelectStatement&)*create_statement.select);
+      if (create_statement.viewColumns) {
+        std::vector<std::shared_ptr<Expression>> projections;
+        ColumnID column_id{0};
+        Assert(create_statement.viewColumns->size() == view->output_column_count(),
+               "Number of Columns in CREATE VIEW does not match SELECT statement");
+        for (const auto& alias : *create_statement.viewColumns) {
```

Comment:
some blank lines for the loops, asserts, ifs would be good here I think

Reply:
You just want to waste paper, don't you? :(

Done. Also added some more comments here.